Reed/Reader/views/ReaderView.swift

```diff
@@ -27,13 +27,24 @@ struct ReaderView: View {
                     data: viewModel.pages,
                     id: \.self,
                     content: { page in
-                        DefinableTextView(
-                            text: .constant(page.content),
-                            tokens: page.tokens,
-                            definerResultHandler: definerResultHandler
-                        )
+                        if viewModel.curPage == -1 || (viewModel.section?.nextNcode != nil && viewModel.curPage == viewModel.pages.endIndex - 1) || (viewModel.section?.prevNcode != nil && viewModel.curPage == 0) {
```

Comment:
Nit: You can line break long compound conditions before the operator.
```
if cond1
    || (subcond1 && subcond2)
    || cond 3
```

Reply:
fixed